app.rb

```diff
@@ -272,7 +272,7 @@ def state_uptimes_between(t1, t2)
         end
       end
       
-      if monitor_time
+      if monitor_time > 0
```

Comment:
if the problem is nil, won't this also error out because nil doesn't have a `>` method?

I'd go with `if !monitor_time.nil?` just to be really explicit. (or if ActiveSupport `if monitor_time.present?`

But if you're confident :shipit: